### ISSUE TITLE
Force int value

### ISSUE THREAD

**Issue by traquall** [surveyjs/editor#142]
### Are you requesting a feature, reporting a bug or ask a queation?

Asking if it exists, if not requesting a feature.

### What is the current behavior?

For every choices the value field is a string

### What is the expected behavior?

I'd like to force the value field to be an int.

To do so, if you can add IDs to modal's fields or add a question option.

Thank you.
Traquall

**Comment by dmitrykurmanov** [comment]
@traquall hello! Maybe I don't understand your use case, but why don't you convert string to integer by  standard way?

**Comment by traquall** [comment]
@dmitrykurmanov Hello 
I can't because the user can put a string like 'item1'

**Comment by dmitrykurmanov** [comment]
@traquall, ok now I understand. you could you number type, please see:

https://plnkr.co/edit/kyE7jikDUTVDGZmldMje?p=preview
https://surveyjs.io/Examples/Library/?id=questiontype-text&platform=Knockoutjs

**Comment by traquall** [comment]
@dmitrykurmanov Thanks, but your examples are not for the survey editor.
I'd like to change the choices type to 'number' but without having to recreate all basic question (radiogroup, checkbox, dropdown, ...)

**Comment by andrewtelnov** [comment]
@traquall Here is the [plunker demo](https://plnkr.co/edit/Zb4HPF?p=preview) that does exactly what you need.
Please notice, it will work with the latest version only. We have added a lot of cool features into new version.

Thank you,
Andrew

**Comment by traquall** [comment]
@andrewtelnov @dmitrykurmanov 
Thank you ! 

Do i need to download the package of the new release ? And do i need to change my knockout script too ? 
I currently use this :
```
<link href="https://surveyjs.azureedge.net/dev/surveyeditor.css" type="text/css" rel="stylesheet" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/knockout/3.3.0/knockout-min.js"></script>
<script src="https://surveyjs.azureedge.net/dev/survey.ko.min.js"></script>
<script src="https://surveyjs.azureedge.net/dev/surveyeditor.min.js"></script>
```

**Comment by andrewtelnov** [comment]
@traquall The current version, released today, are 0.12.24
```
<script src="https://surveyjs.azureedge.net/0.12.24/survey.ko.js"></script>
<link href="https://surveyjs.azureedge.net/0.12.24/surveyeditor.css" type="text/css" rel="stylesheet" />
<script src="https://surveyjs.azureedge.net/0.12.24/surveyeditor.js"></script>
```
I would not recommend to use the dev version. It is mostly for testing.

Thank you,
Andrew
